cves/CVE-2015-8217.yml

```diff
@@ -96,8 +100,8 @@ fixes:
 vccs:
 - commit: c8dd048ab8cff815c9f4b16a62db0b74df011f0a
   note: Identified by archeogit,
-- commit: ''
-  note: ''
+- commit: 064698d381e1e7790f21b0199a8930ea04e2e942
+  note: Identified by archeogit,
```

Comment:
You added this VCC, so it wasn't "Identified by archeogit" -- how did you find this VCC and how did you decide it was a VCC (be brief).